src/config/layers.js

```diff
@@ -534,7 +536,7 @@ export const netzkarteShowcasesLight = new TrafimageMapboxLayer({
 
 export const netzkarteShowcasesNetzkarte = new TrafimageMapboxLayer({
   name: 'ch.sbb.netzkarte.layer',
-  key: 'ch.sbb.netzkarte',
+  key: 'ch.sbb.netzkarte.layer',
```

Comment:
Why `.layer`? Old permalinks could stop working.

Reply:
For consistency, to have similar key and name, but we can keep the key without '.layer' sure :)